contexts-config-store/store/persistence/default_contexts_config.go

```diff
@@ -0,0 +1,28 @@
+package persistence
+
+import (
+	"github.com/google/uuid"
+	api "github.com/kurtosis-tech/kurtosis/contexts-config-store/api/golang"
+	"github.com/kurtosis-tech/kurtosis/contexts-config-store/api/golang/generated"
+	"github.com/kurtosis-tech/stacktrace"
+	"strings"
+)
+
+const (
+	defaultContextName = "default"
+)
+
+func NewDefaultContextsConfig() (*generated.KurtosisContextsConfig, error) {
+	// This logic is similar to uuid_generate.GenerateUUIDString, but we don't want to pull container engine lib here
+	// just for this.
+	// TODO: extract the UUID generator outside of container engine lib maybe
+	randomUuid, err := uuid.NewRandom()
+	if err != nil {
+		return nil, stacktrace.Propagate(err, "Unable to generate a random UUID for the default context")
+	}
+	newContextUuidStr := strings.Replace(randomUuid.String(), "-", "", -1)
```

Comment:
perhaps take these magic constants out

Reply:
if we are in the business of generating uuids should we generate uuids for other additions too?

weird that default would look like "aedf1230...." and someone can just put in "hello"